### DISCUSSION TITLE
Absolutely stunning

### DISCUSSION THREAD

**Discussion by diraneyya** [martin-t/rec-wars#6] [Show and tell]
I have to say this is absolutely stunning what you have done.

I am very interested in the `gl.js` file you created. As I understand, you have created your own graphics runtime. Where do you import those functions in the Rust code and where is the `gl.js` file developed? Is it in a private repository?

PS: I can see where the imported functions are. They are in the `importObject`. I am speechless!

PS2: After checking the RUST code, it does not seem as if the functions in the `importObject` are used directly. I am starting to believe that there are missing pieces here. Can you help?

PS3: I found it! It is in the miniquad repo on your GitHub account!!

Regards.

**Comment by martin-t** [comment]
Yep, `gl.js` is from the macroquad engine which internally uses miniquad. The repo on my account is just a fork, not my work at all.
